example/package.json

```diff
@@ -69,7 +69,6 @@
     "jasmine-reporters": "^2.3.2",
     "js-yaml": "^3.13.1",
     "kes": "^2.2.5",
-    "lodash.assignin": "^4.2.0",
```

Comment:
As I was removing now-unused code, there were a couple packages that we are no longer using.